From 0074eb1dcc77bb08f081b8c753b5dbe2f8e77f93 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Tue, 07 Aug 2018 16:40:22 +0300
Subject: [PATCH] board [dali]: add support for fw flashing cleanup

Remove external FW of PCIe PHY since it is used from the embedded
copy inside the BTL image
---

diff --git a/board/habanalabs/dali/post-image.sh b/board/habanalabs/dali/post-image.sh
index 988a4e7..3cb7f88 100755
--- a/board/habanalabs/dali/post-image.sh
+++ b/board/habanalabs/dali/post-image.sh
@@ -42,17 +42,12 @@
 	# Used by TRACE32 (Lauterbach JTAG)
 	dd status=none conv=sync bs=4 if=u-boot.bin						 > dali-u-boot.bin
 
-	# Swap FW
-	swap_file btl/pcie_phy_fw.bin pcie_phy_fw.bin.be
-	PCIE_PHY_FW_BINFILE=pcie_phy_fw.bin.be
-
 	### Construct bin file
 	# BTL likes these to be 16B aligned in size
 	dd status=none conv=sync bs=16 if=preboot.bin						 > dali-preboot.bin
 	# prepare BTL image
 	BTL_CMD_ARGS="${BTL_CMD_ARGS} SPL_SIZE=$(printf "0x%X\n" `stat -c "%s" ${SPL_BINFILE}`)"
 	BTL_CMD_ARGS="${BTL_CMD_ARGS} UBOOT_SIZE=$(printf "0x%X\n" `stat -c "%s" ${UBOOT_BINFILE}`)"
-	BTL_CMD_ARGS="${BTL_CMD_ARGS} PCIE_PHY_FW_SIZE=$(printf "0x%X\n" `stat -c "%s" ${PCIE_PHY_FW_BINFILE}`)"
 	cp -a ${BOARD_BASE}/btl .
 	make -C btl clean all ${BTL_CMD_ARGS}
 	cp btl/sectors-be.bin .
@@ -79,8 +74,6 @@
 	dd status=none conv=sync bs=64K		if=${SPL_BINFILE}				>> dali-spi-flash.bin.nocrc
 	# U-Boot
 	dd status=none conv=sync bs=1M		if=${UBOOT_BINFILE}				>> dali-spi-flash.bin.nocrc
-	# PCIe FW
-	dd status=none conv=sync bs=64K		if=${PCIE_PHY_FW_BINFILE}			>> dali-spi-flash.bin.nocrc
 
 	# Prepare checksum
 	CKSUM=`cksum dali-spi-flash.bin.nocrc | cut -d' ' -f1`
